src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/crud/tutorial-group-form/tutorial-group-form.component.ts

```diff
@@ -152,7 +152,11 @@ export class TutorialGroupFormComponent implements OnInit, OnChanges, OnDestroy
         } else if (!originalHasSchedule && !updateHasSchedule) {
             return false;
         } else {
-            return !isEqual({ ...this.form.value.schedule }, this.existingScheduleFormDate);
+            const a = { ...this.form.value.schedule };
+            delete a.location;
+            const b = { ...this.existingScheduleFormDate };
+            delete b.location;
+            return !isEqual(a, b);
```

Comment:
What are an and b here? The old and new tutor group?

Reply:
Renamed them in 39f98211b8927199703dcdef58fc18707c49b11a